Products/ZenCollector/daemon.py

```diff
@@ -584,6 +584,10 @@ def _updateConfig(self, cfg):
             # all pending tasks have completed
             if not self.options.cycle:
                 self._pendingTasks.append(taskName)
+        # put tasks on pause after configuration update to prevent unnecessary collections ZEN-25463
+        for devId in self._unresponsiveDevices:
```

Comment:
This seems excessive. This method is for updating one config and the for loop is modifying every task that is paused.